app/shop.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+from app.product_calculation import product_calculation_print
+from app.movement import trip_to_the_store
+from app.movement import trip_to_the_home
+
+
+def purchase(count_milk: int,
+             count_bread: int,
+             count_butter: int,
+             customer,
+             shop):
+    """
+    Trip to the store (change of coordinates)
+    """
+    trip_to_the_store(customer, shop)
+
+    """
+    Receipt printing
+    """
+    date = datetime(2021, 1, 4, 12, 33, 41).strftime("%d/%m/%Y %H:%M:%S")
```

Comment:
use datetime.datetime.now()

Reply:
I did this at first, but then I changed it, I thought that the tests might not pass

Reply:
I'll fix everything now